src/auth/ui/vue/show.ts

```diff
@@ -712,20 +721,41 @@ const Panel = VueWebview.compilePanel(AuthWebview)
 let activePanel: InstanceType<typeof Panel> | undefined
 let subscriptions: vscode.Disposable[] | undefined
 
+/**
+ * Different places the Add Connection command could be executed from.
+ *
+ * Useful for telemetry.
+ */
 export const AuthSources = {
     addConnectionQuickPick: 'addConnectionQuickPick',
     firstStartup: 'firstStartup',
     codecatalystDeveloperTools: 'codecatalystDeveloperTools',
-    unknown: 'unknown',
+    vscodeExecution: vscodeExecution,
     cwQuickPick: cwQuickPickSource,
-    cwTreeNode: cwTreeNodeSource
+    cwTreeNode: cwTreeNodeSource,
 } as const
 
-export type AuthSource = typeof AuthSources[keyof typeof AuthSources]
+export type AuthSource = (typeof AuthSources)[keyof typeof AuthSources]
+
+export const showManageConnections = Commands.declare(
+    { id: showConnectionsPageCommand, compositeKey: { 1: 'source' } },
+    (context: vscode.ExtensionContext) => (_: VsCodeCommandArg, source: AuthSource, serviceToShow?: ServiceItemId) => {
+        // The auth webview page does not make sense to use in C9,
+        // so show the auth quick pick instead.
+        if (isCloud9('any')) {
+            return addConnection.execute()
+        }
 
-export function isAuthSource(obj: any): obj is AuthSource {
-    return Object.values(AuthSources).includes(obj)
-} 
+        if (_ !== placeholder) {
```

Comment:
if we are actually using the parameter we should give it a name. `_` usually mean "ignore"

Reply:
This handles the edge case which the docstring on the type `VsCodeCommandArg` should hopefully explain.

Did you have a name in mind?

Reply:
`node` is what we usually call it